profiles/common/modules/custom/nexteuropa_ecl_migrate/nexteuropa_ecl_migrate.module

```diff
@@ -180,3 +181,10 @@ function _nexteuropa_ecl_migrate_dt_menus() {
 
   module_invoke('context', 'save', $context);
 }
+
+/**
+ * Custom function that moves the proper blocks in the right areas.
+ */
+function _nexteuropa_ecl_migrate_europa_main_menu() {
+  multisite_drupal_toolbox_add_block_context('site_wide', 'main-menu', 'system', 'main-menu', 'navigation');
```

Comment:
In the europa.info, the first one is navigation

; ========================================
; Regions
; ========================================
regions[navigation] = Navigation